src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -21002,7 +21002,7 @@ public void M()
         }
 
         [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/56256")]
-        public void MemberNotNull_LocalFunction_BaseField()
+        public void MemberNotNull_NonStaticLocalFunction_BaseField()
```

Comment:
Is there any test which is for 'MemberNotNull' on a local function within a constructor?